prowler/providers/gcp/services/kms/kms_key_rotation_enabled/kms_key_rotation_enabled.py

```diff
@@ -11,18 +13,31 @@ def execute(self) -> Check_Report_GCP:
             report.resource_id = key.id
             report.resource_name = key.name
             report.location = key.location
-            report.status = "FAIL"
-            report.status_extended = (
-                f"Key {key.name} is not rotated every 90 days or less."
-            )
+            now = datetime.datetime.now()
+            condition_next_rotation_time = False
+            if key.next_rotation_time:
+                next_rotation_time = datetime.datetime.strptime(
+                    key.next_rotation_time, "%Y-%m-%dT%H:%M:%SZ"
+                )
+                condition_next_rotation_time = (
+                    abs((next_rotation_time - now).days) <= 90
+                )
+            condition_rotation_period = False
             if key.rotation_period:
-                if (
+                condition_rotation_period = (
                     int(key.rotation_period[:-1]) // (24 * 3600) <= 90
-                ):  # Convert seconds to days and check if less or equal than 90
-                    report.status = "PASS"
-                    report.status_extended = (
-                        f"Key {key.name} is rotated every 90 days or less."
-                    )
+                )
+            if condition_rotation_period and condition_next_rotation_time:
+                report.status = "PASS"
+                report.status_extended = f"Key {key.name} is rotated every 90 days or less and the next rotation time is in less than 90 days."
+            else:
+                report.status = "FAIL"
+                if condition_rotation_period:
+                    report.status_extended = f"Key {key.name} is rotated every 90 days or less but the next rotation time is in more than 90 days."
```

Comment:
How a key can have a rotation period of 90 days and its next rotation time in more than 90 days?

Reply:
In the `Key Rotation` options, you need to select both `Key Rotation Period` and `Starting On`. If you choose a Starting On date that is more than 90 days in the future, the key will not be rotated for the first 90 days of the initial version.
<img width="522" alt="image" src="https://github.com/user-attachments/assets/95b8a904-018f-4a14-ae5e-52a26c043bd8">

Reply:
Just the info that I needed, thank you @JOSHUAJEBARAJ !